iterators.tex

```diff
@@ -6442,56 +6461,84 @@
 container copies the elements, which cannot be done in constant time.
 \exitexample
 
-\begin{codeblock}
+\begin{itemdecl}
 @\newtxt{template <class T>}@
-@\newtxt{struct is_view;}@
+@\newtxt{struct enable_view \{ \};}@
 
 @\newtxt{struct view_base \{ \};}@
 
+@\newtxt{template <class T>}@
+@\newtxt{constexpr bool \xname{view_predicate}; // \expos}@
```

Comment:
> Also, `__view_predicate` is a strange name. Why not `__is_view`?

Mostly to dissociate from the old trait and the connotations of naming something `is_foo`. `<type_traits>` has taught us that `is_foo` is a useful concept-membership trait; using a non-`is` name avoids the inevitable questions about turning it into a useful user-exposed trait. That said, I of course have no strong feelings about the naming of something that is `\expos`. Shall I change it?